README.md

```diff
@@ -117,6 +117,18 @@ disable this:
 let g:test#preserve_screen = 1
 ```
 
+### Quickfix Strategies
+If you want your test results to appear in the quickfix window, use one of the 
+following strategies:
+
+ * Make
+ * Neomake
+ * MakeGreen
+ * Dispatch.vim
+
+When Dispatch.vim is installed, it will be used to determine the compiler and 
+errorformat settings for the test runner.
```

Comment:
I would prefer for this paragraph to be something like:

> Even if you're using a quickfix strategy other than Dispatch.vim, it's still recommended you have Dispatch.vim installed, as the strategies will automatically use it to find and apply the correct Vim compiler, ensuring the test output is correctly parsed for the quickfix window.

There is no need to mention `errorformat` when we already mention "compiler", as setting `errorformat` is the only thing the compiler does (other than setting `makeprg` which we don't use).